build.gradle.kts

```diff
@@ -250,6 +250,8 @@ koverMerged {
                 excludes += "io.element.android.appnav.loggedin.LoggedInPresenter$*"
                 // Some options can't be tested at the moment
                 excludes += "io.element.android.features.preferences.impl.developer.DeveloperSettingsPresenter$*"
+                // Temporary until we have actually something to test.
+                excludes += "io.element.android.features.pin.impl.*Presenter"
```

Comment:
Will introduce `KoverIgnored` annotation in a seperate PR to manage all these exclusions more properly.